tasks/copy_to_archive/copy_to_archive.py

```diff
@@ -253,7 +253,7 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
         # post to metadata SQS for each granule
         sqs_library.post_to_metadata_queue(sqs_body, metadata_queue_url)
 
-    return {"granules": granules_list, "copied_to_glacier": copied_file_urls}
+    return {"granules": granules_list, "copied_to_archive": copied_file_urls}
```

Comment:
Thinking this should be `copied_to_orca` as this is forward facing to the user and will better define what was sent to ORCA.